faq.md

```diff
@@ -290,9 +290,18 @@ Yes, a _manufacturer_ can also be an _open-source software steward_, but it cann
 
 <details>
     <a name="q-what-is-a-harmonized-standard-and-why-does-it-matter"></a>
-    <summary><strong><a name="faq-tmp-56" href="#faq-tmp-56">tmp-56.</a> What is a <em>harmonised standard</em> and why does it matter?</strong></summary>	
+    <summary><strong><a name="faq-tmp-56" href="#faq-tmp-56">tmp-56.</a> What is a <em>harmonised standard</em> and why does it matter?</strong></summary>
 
-A _harmonised standard_ is a standard developed by the [European Standardisation Organisations (ESOs)][ESO]. It takes the legal requirements of the law and translates them into technical requirements for those implementing it. Certain _harmonised standards_ provide organisations which implement them with a _presumption of conformity_ with the law. This is why it is expected that most organisations will choose to implement the _harmonised standards_ to comply with the CRA. The ORC WG maintains a [list of _harmonised standards_][standards] requested by the European Commission to the ESOs.
+A _harmonised standard_ is a standard adopted by one of the [European Standardisation Organisations (ESOs)][ESO]. Certain (but not all) _harmonised standards_ are referenced in the _Official Journal of the European Union_.
+Any harmonised standard  referenced in this way provides organisations that implement them with a _presumption of conformity_ with the law.
```

Comment:
```suggestion
Harmonised standards referenced in this way provide organisations that implement them with a _presumption of conformity_ with the regulation. Harmonised standards may be referenced with restrictions, in which case they only provide partial _presumption of conformity_.
```

Reply:
I do not think this is entirely correct  as it also applies to directives (see e.g. https://eur-lex.europa.eu/eli/dec_impl/2025/138/oj).
The NLF recitals say "The presumption of conformity to a legal provision conferred by conformity to a harmonised standard should enhance recourse to compliance with harmonised standards."

Reply:
See [`c2e9082` (#188)](https://github.com/orcwg/cra-hub/pull/188/commits/c2e9082ede5830eccc567e24bd17c3b61293d4a2)